src/zoomifytilesource.js

```diff
@@ -0,0 +1,113 @@
+(function($) {
+
+    /**
+     * @class ZoomifyTileSource
+     * @classdesc A tilesource implementation for the zoomify format.
+     *
```

Comment:
I agree with @avandecreme we should include a link to the format. This is probably a good spot. Also please describe how to format a tileSource object so that Viewer.open will open it as a ZoomifyTileSource. This would also be a good place to explain that we don't yet support the XML version.